src/components/indicator-message/index.test.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { ZnsLink } from '@zer0-os/zos-component-library';
+import IndicatorMessage from '.';
+
+describe('Indicator new message', () => {
+  const subject = (props: any = {}) => {
+    const allProps = {
+      countNewMessage: 0,
+      ...props,
+    };
+
+    return shallow(<IndicatorMessage {...allProps} />);
+  };
+
+  it('adds className', () => {
+    const wrapper = subject({ className: 'newMessages' });
+
+    expect(wrapper.hasClass('newMessages')).toBe(true);
+  });
+
+  it('renders indicator when new messages arrive', () => {
+    const wrapper = subject({ countNewMessage: 0 });
+
+    expect(wrapper.find('.indicator__new-message').exists()).toBe(false);
+
+    wrapper.setProps({ countNewMessage: 2 });
+
+    expect(wrapper.find('.indicator__new-message').exists()).toBe(true);
+  });
+
+  it('it should not renders indicator when click on it', () => {
+    const closeIndicatorSpy = jest.fn();
+    const wrapper = subject({ countNewMessage: 3, closeIndicator: closeIndicatorSpy });
+
+    expect(wrapper.find('.indicator__new-message').exists()).toBe(true);
+
+    wrapper.find('.indicator__new-message-bar').simulate('click');
+
+    expect(wrapper.prop('closeIndicator')).toHaveBeenCalled;
```

Comment:
pretty sure that `toHaveBeenCalled` is a function. this expectation probably isn't doing what we think it is, and might actually be causing problems. was this the branch that had timeout failures? this could be the reason! it might be waiting for an expectation, and timing out when none is there.

Reply:
No, the reason was the setInterval function which caused the problem, I fixed the problem, i added a 

> `beforeEach(() => {

     jest.useFakeTimers();
   })` 
in the test file to fix the issue

Reply:
ah. k. so, two takeaways:
- the first is just that. .. my main point was that this test probably isn't working. `toHaveBeenCalled`  is a function, and we're not calling it here. (we're also accessing it indirectly using `prop` instead of just having the expectation based on the spy itself. let's change that while we're at it.)
- second is that components are generally the wrong place for code that uses setInterval. there are exceptions, but if you find yourself using fakeTimers in a component test, there's probably a better way to organize the code. i'll add comments elsewhere to make it clearer what i'm talking about for this one.